tests/drivers/clock_control/nrf_clock_control/src/main.c

```diff
@@ -9,13 +9,17 @@
 #include <zephyr/drivers/clock_control/nrf_clock_control.h>
 #include <zephyr/kernel.h>
 #include <zephyr/ztest.h>
+#if defined(CONFIG_CLOCK_CONTROL_NRF2_AUXPLL)
+#include <zephyr/dt-bindings/clock/nrf-auxpll.h>
+#endif
```

Comment:
include guards not neccesary :) It does not hurt to include them unconditionally